src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const KEYS = Object.values(robot);
+  const VALUES = Object.keys(robot);
+  const INVERSED_ROBOT = {};
+  const UNIQUE_KEYS = [ ...new Set(KEYS) ];
+
+  if (UNIQUE_KEYS.length !== VALUES.length) {
+    return null;
+  }
+
+  for (let i = 0; i < KEYS.length; i++) {
```

Comment:
and use `Object.entries()` to iterate through the key-value pairs of robot, it will make code more simple